tests/foreman/ui_airgun/test_syncplan.py

```diff
@@ -473,3 +523,107 @@ def test_positive_synchronize_custom_product_weekly_recurrence(
         repo_values = session.repository.read(product.name, repo.name)
         for repo_type in ['Packages', 'Errata', 'Package Groups']:
             assert repo_values['content_counts'][repo_type] > '0'
+
+
+@tier3
+def test_positive_synchronize_custom_product_custom_cron_real_time(
+        session, module_org):
+    """Create a sync plan with real datetime as a sync date,
+    add a custom product and verify the product gets synchronized
+    on the next sync occurrence based on custom cron interval
+
+    :id: c551ef9a-6e5a-435a-b24d-e86de203a2bb
+
+    :expectedresults: Product is synchronized successfully.
+
+    :CaseLevel: System
+    """
+    plan_name = gen_string('alpha')
+    product = entities.Product(organization=module_org).create()
+    repo = entities.Repository(product=product).create()
+    with session:
+        startdate = session.browser.get_client_datetime()
+        # calculating next sync based on (*/5 * * * *) will have occurrences (0, 5, 10, 15..)
+        next_sync = (5 - int(startdate.strftime('%M')) % 5) * 60
+        session.syncplan.create({
+            'name': plan_name,
+            'interval': SYNC_INTERVAL['custom'],
+            'cron_expression': '*/5 * * * *',
+            'description': 'sync plan create with start time',
+            'date_time.start_date': startdate.strftime("%Y-%m-%d"),
+            'date_time.hours': startdate.strftime('%H'),
+            'date_time.minutes': startdate.strftime('%M'),
+        })
+
+        assert session.syncplan.search(plan_name)[0]['Name'] == plan_name
+        session.syncplan.add_product(plan_name, product.name)
+        # Waiting part of delay and check that product was not synced
+        time.sleep(next_sync/2.5)
+        with raises(AssertionError) as context:
+            validate_task_status(repo.id, max_tries=2)
+        assert 'No task was found using query' in str(context.value)
+        validate_repo_content(
+            repo, ['erratum', 'package', 'package_group'], after_sync=False)
+        # Waiting part of delay that left and check that product was synced
+        time.sleep(next_sync/2.5)
+        validate_task_status(repo.id, repo_backend_id=repo.backend_identifier)
+        validate_repo_content(repo, ['erratum', 'package', 'package_group'])
+        repo_values = session.repository.read(product.name, repo.name)
+        for repo_type in ['Packages', 'Errata', 'Package Groups']:
+            assert repo_values['content_counts'][repo_type] > '0'
```

Comment:
for more accuracy,  we have to compare numbers not strings ('00' > '0' is True):
```python
assert int(repo_values['content_counts'][repo_type]) > 0
```